docs/integrations/ai-engines/monkeylearn.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: MindsDB and MonkeyLearn
+sidebarTitle: MonkeyLearn
+---
+
+MonkeyLearn is a No-code text analysis tool. MindsDB allows you to use pre-built & custom MonkeyLearn models to use its features like classifying text according to user needs and fields of interest like business, reviews, comments, and customer feedback.
+
+## How to bring MonkeyLearn Models to MindsDB
+
+Before creating a model, you will need to create the ML_ENGINE for MonkeyLearn using the `CREATE ML_ENGINE` syntax
+
+```sql
+Create ML_ENGINE monkeylearn
+FROM monkeylearn
+```
+
+<p align="center">
+  <img src="/docs/assets/tutorials/monkeylearn/1.create_ml.png" />
```

Comment:
Please remove `/docs` from each image link. It should be like `/assets/tutorials/monkeylearn/image_name.png`